Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contests/contest-details/ContestDetailsPage.tsx

```diff
@@ -106,6 +105,10 @@ const ContestDetailsPage = () => {
             >
                 Full Results
             </Button>
+            <AdministrationLink
+              to={`/${PROBLEMS_PATH}?filter=contestid~equals~${contestId}%26%26%3Bisdeleted~equals~false&sorting=id%3DDESC`}
+              text="Problems"
+            />
```

Comment:
Can we use the tab-link in the contest's Edit page instead of the Problems grid main page?

Reply:
Fixed to `/${CONTESTS_PATH}/${contestId}#tab-problems